c++/src/kj/compat/tls-test.c++

```diff
@@ -914,6 +914,11 @@ public:
     receiver = tlsServer.wrapPort(kj::mv(baseReceiverPtr));
   }
 
+  TlsReceiverTest(TlsReceiverTest&&) = default;
```

Comment:
I don't think it would be. We're specifying the same information in a few more lines, and for our trouble we get better compiler errors if we do see an attempted copy.